docs/reference/modules/scripting/using.asciidoc

```diff
@@ -110,12 +110,12 @@ second version is only compiled once.
 [[modules-scripting-file-scripts]]
 === File-based Scripts
 
-To increase security, non-sandboxed languages can only be specified in script
-files stored on every node in the cluster.  File scripts must be saved in the
-`scripts` directory whose default location depends on whether you use  the
+To increase security, scripts can only be specified in files stored on every
```

Comment:
This line isn't really true now, eg expressions and painless are inline:true by default.  Perhaps we should continue to describe languages as sandboxed (or maybe "safe"?) to distinguish painless, expressions, mustache from all the rest?